### ISSUE TITLE
stdin keep growing with input from all the POST 

### ISSUE THREAD

**Issue by postak** [fnproject/fdk-node#19]
Hi,

I have a simple func.js

```
var fdk=require('@fnproject/fdk');
  fdk.handle( function(input) {
  return input
})
```

the function continue to append the input data from previous call  until the docker is running (I use http-stream ) 

```
postak@mbp13 curl -X POST -d @payload.json 'http://127.0.0.1:8080/t/myapp/myfunc
"ciao"postak@mbp13 curl -X POST -d @payload.json 'http://127.0.0.1:8080/t/myapp/myfunc
"ciaociao"postak@mbp13 curl -X POST -d @payload.json 'http://127.0.0.1:8080/t/myapp/myfunc
"ciaociaociao"postak@mbp13 curl -X POST -d @payload.json 'http://127.0.0.1:8080/t/myapp/myfunc
"ciaociaociaociao"postak@mbp13 curl -X POST -d @payload.json 'http://127.0.0.1:8080/t/myapp/myfunc
```

this is my func.yaml

```
schema_version: 20180708
name: myfunc
version: 0.0.66
runtime: node
entrypoint: node func.js
format: http-stream
triggers:
- name: myfunc
  type: http
  source: /myfunc
```

**Comment by zootalures** [comment]
This should now be fixed in 0.0.11, can you retest and close if it’s fixed ?

**Comment by postak** [comment]
fixed
